### DISCUSSION TITLE
2024-08-05 월요일

### DISCUSSION THREAD

**Discussion by Captain-Kim** [nbc-final-a10/TripBuddy#198] [기록보드]
2024-08-05 월요일 기록보드입니다.

## 여행가고십조(A10) 일일 공통 일과표

| 시간 | 일과 |  기록 |
| :-: | :-: | :-: |
| 09:00 ~ 10:00 | 아침 회의 | |
| 10:00 ~ 11:00 | MVP 발표 준비 | |
| 11:00 ~ 13:00 | 집중 개발 time | |
| 13:00 ~ 14:00 | 점심 식사 | |
| 14:00 ~ 16:00 | MVP 발표 | |
| 16:00 ~ 18:00 | 집중 개발 TIME | |
| 18:00 ~ 19:00 | 저녁 식사 | |
| 19:00 ~ 20:00 | 저녁 회의 | |
| 20:00 ~ 21:00 | 5분 기록보드 작성 / 개인 회고 | |

```
## 개인 기록보드 서식 (김병준)

### 오늘 한 일

- 내용

### 트러블 슈팅

- 내용

### 내일 계획

- 내용
```

```
// 이미지 테이블
<table>
  <tr>
    <th>이미지 1번 제목</th>
    <th>이미지 2번 제목</th>
  </tr>
  <tr>
    <td><img src="이미지 주소" width="300" height="300"/></td>
    <td><img src="이미지 주소" width="300" height="300"/></td>
  </tr>
</table>
```

**Comment by eunohhh** [comment]
## 개인 기록보드 서식 (오은)

### 오늘 한 일

- 스토리 여러개 겹쳐서 렌더링 되도록 처리
- 기본 언어 한글로
- 스토리 높이 모바일에 맞추어 수정

### 트러블 슈팅

- 없음

### 내일 계획

- 스토리 완성
